numpy/core/src/npymath/npy_math_internal.h.src

```diff
@@ -635,11 +711,14 @@ npy_divmod@c@(@type@ a, @type@ b, @type@ *modulus)
     @type@ div, mod, floordiv;
 
     mod = npy_fmod@c@(a, b);
```

Comment:
The current failures in CI are because of this line. The CI in windows seems to be using GCC-8. I tried with GCC-8 and GCC-9 and on both, this line sets the invalid flag and raises error on python side. on GCC-5.4 (which was my default) it didnt set a flag and just set the result to nan. Looking for advice here, should we try to workaround the compiler behavior for GCC-8 and GCC-9, to stick to existing numpy behavior which we have for earlier compiler versions ? CON: Makes code slightly more convoluted. PRO: Will provide uniform behavior across different numpy builds.
( We already went down this path since we are trying to make it consistent on clang with the npy_set_floatstatus_divbyzero changes)

Reply:
i have added nan checks, so that this provides uniform builds, atleast for the CI.